persistent-data-provider/schema.graphql

```diff
@@ -76,12 +95,15 @@ enum SortBusinessesBy {
   address_DESC
   lastUpdated_ASC
   lastUpdated_DESC
+  businessInferences_ASC
+  businessInferences_DESC
```

Comment:
Why do we need ordering on the server side?  I'd figure ordering would only matter on the server side you limiting a query result to a certain number of TOP results.  Otherwise the client can do the ordering.

Reply:
This file is autogenerated by Keystone. Even if we removed this functionality, it would be added again whenever we build the schema.